docs/use_cases/q_and_a/rag_cli.md

```diff
@@ -58,6 +59,35 @@ Here are some high level steps to get you started:
    ```
 1. **Open a Chat REPL**: You can even open a chat interface within your terminal! Just run `$ llamaindex-cli rag --chat` and start asking questions about the files you've ingested.
 
+### Create a LlamaIndex chat application
+
+You can also create a full-stack chat application with a FastAPI backend and NextJS frontend using your embedded data.
+
+To bootstrap the application, make sure you have NodeJS and npx installed on your machine. If not, please refer to the [LlamaIndex.TS](https://ts.llamaindex.ai/getting_started/installation) documentation for instructions.
+
+Once you have everything set up, creating a new application is easy. Simply run the following command:
+
+`$ llamaindex-cli rag --create-llama`
+
+It will call our `create-llama` package so you will need to provide several pieces of information in the to create the app, you can find more information about the `create-llama` package on [npmjs - create-llama](https://www.npmjs.com/package/create-llama#example)
```

Comment:
```suggestion
It will call our `create-llama` tool, so you will need to provide several pieces of information to create the app. You can find more information about the `create-llama` on [npmjs - create-llama](https://www.npmjs.com/package/create-llama#example)
```